pkg/minikube/sshagent/sshagent.go

```diff
@@ -106,6 +106,9 @@ func isRunning(cc *config.ClusterConfig) (bool, error) {
 func Stop(profile string) error {
 	cc, err := config.Load(profile)
 	if err != nil {
+		if config.IsNotExist(err) {
+			return nil
```

Comment:
ideally it would return an ErrorType here that the caller of "Stop" will check against it and decide what to do with it. but this is still acceptable

Reply:
Implemented